src/main/java/org/kohsuke/github/GHGistFile.java

```diff
@@ -76,4 +100,14 @@ public String getContent() {
     public boolean isTruncated() {
         return truncated;
     }
+
+
+    /**
+     * set the owner object of the GHGistFile.
+     *
+     * @param owner the object stores owner information
+     */
+    public void setOwner(GHUser owner) {
+        this.owner = owner;
+    }
```

Comment:
> Sorry, why add this?

Sorry, the reason I am adding this setter is that I want to pass the `owner` object from the `GHGist` object to each `GHGistFile` object so that get content can use `owner.root()` to create request. At a second thought, probably defining a `wrapUp` method to set the owner as the `GHGist` object owning this current `GHGistFile` object would make more sense. I have make this revision accordingly. Thank you again for your suggestions.